ani-cli

```diff
@@ -325,9 +334,9 @@ play() {
 
 # setup
 agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/121.0"
-allanime_refr="https://allanime.to"
+allanime_refr="https://allmanga.to"
 allanime_base="allanime.day"
-allanime_api="https://api.${allanime_base}"
+allanime_api="https://api.${allanime_base}" #api is down apparently
```

Comment:
Is it still down?

Reply:
> Is it still down?

![image](https://github.com/user-attachments/assets/d20da9fa-b68a-4e63-bd06-8810e01c3fb4)
fixed

Reply:
lemme change that rq